src/saving/SaveManagerGUI.cs

```diff
@@ -166,17 +172,39 @@ private void DeleteSelectedButtonPressed()
         deleteSelectedConfirmDialog.PopupCenteredMinsize();
     }
 
+    private void DeleteOldButtonPressed()
+    {
+        deleteOldConfirmDialog.DialogText =
+            "Deleting all old Auto and Quick saves cannot be undone, " +
+            "are you sure you want to permanently delete the following?\n" +
+            $" >>> {Mathf.Clamp(SaveHelper.CountSaves("auto_save").count - 1, 0, Settings.Instance.MaxAutoSaves)} " +
```

Comment:
I think it's fine to get the fresh counts to make sure that up to date info is shown to the user.
This math should be outside the text formatting. Also isn't Mathf always with floats? Whereas Math.Clamp(int, int) would be a method working on ints.

Reply:
I tried to replace Mathf.Clamp with Math.Clamp but it turns out Math does not have that member function. I looked up Microsoft docs and it says the Math.Clamp API is non CLS-Compliant. Does that have something to do with Visual Studio not recognising the API?

Reply:
Actually it looks like we have our own Clamp: https://github.com/Revolutionary-Games/Thrive/blob/0e79166eedcc408980e2215b63cba2c37acb926b/src/general/MathUtils.cs#L12
Which should work like `(SaveHelper.CountSaves("auto_save").count - 1).Clamp(0, Settings.Instance.MaxAutoSaves)`

Reply:
Ah I see, thanks.